app/routes/search/agendaitems.js

```diff
@@ -69,6 +68,13 @@ export default class AgendaitemSearchRoute extends Route {
         entry.shortTitle = entry.shortTitle.replaceAll(highlightShortTitleEmsRemoved, agendaitem.highlight.shortTitle[i]);
       }
     } 
+    if ((agendaitem.highlight && agendaitem.highlight.title) && agendaitem.highlight.title[0]?.split(" ").length < agendaitem.attributes.title?.split(" ").length ) {
```

Comment:
would make sense to make this behavior a function since you are doing exactly the same for the `title` as well as `shortTitle`